_posts/2022-03-07-ssdt-dacpac-netcore.md

```diff
@@ -48,4 +48,6 @@ In order to build the project with `dotnet` use:
 dotnet build /p:NetCoreBuild=true
 ```
 
+Please note that using the SDK will no longer allow you to build and publish the Database project through Visual Studio. Please use the command line tools for this purpose.
```

Comment:
Suggested change:
>> Please note that using the SDK may cause build and publish through Visual Studio to fail. Please use the command line tools for this. In a future update to Visual Studio, this will be fixed.